08-configs.md

```diff
@@ -1,46 +1,50 @@
 ## Configs top-level element
 
-Configs allow services to adapt their behaviour without the need to rebuild a Docker image. Configs are comparable to Volumes from a service point of view as they are mounted into service's container's filesystem. The actual implementation detail to get the configuration provided by the platform can be set from the Configuration definition.
+Configs allow services to adapt their behaviour without the need to rebuild a Docker image.
 
-When granted access to a config, the config content is mounted as a file in the container. The location of the mount point within the container defaults to `/<config-name>` in Linux containers and `C:\<config-name>` in Windows containers.
+Services can only access configs when explicitly granted by a [`configs`](05-services.md#configs) attribute.
 
-By default, the config must be owned by the user running the container command but can be overridden by service configuration.
-By default, the config must have world-readable permissions (mode 0444), unless service is configured to override this.
+As with volumes, configs are mounted as files into a service's container's filesystem. The location of the mount point within the container defaults to `/<config-name>` in Linux containers and `C:\<config-name>` in Windows containers. 
 
-Services can only access configs when explicitly granted by a [`configs`](05-services.md#configs) subsection.
+By default, the config:
+- Is owned by the user running the container command but can be overridden by service configuration.
+- Has world-readable permissions (mode 0444), unless the service is configured to override this.
 
-The top-level `configs` declaration defines or references
-configuration data that can be granted to the services in this
-application. The source of the config is either `file` or `external`.
+The top-level `configs` declaration defines or references configuration data that is granted to services in your Compose application. The source of the config is either `file` or `external`.
 
 - `file`: The config is created with the contents of the file at the specified path.
-- `external`: If set to true, specifies that this config has already been created. Compose does not
+- `external`: If set to true, `external` specifies that this config has already been created. Compose does not
   attempt to create it, and if it does not exist, an error occurs.
-- `name`: The name of the config object on Platform to lookup. This field can be used to
+- `name`: The name of the config object in Docker to lookup. This field can be used to
```

Comment:
Check: is this change correct? This is how it is in secrets.md

Reply:
WDYT?
```suggestion
- `name`: The name of the config object in Container engine to lookup. This field can be used to
```

Reply:
agreed, changed!